Tutorial/Arks Road/Card Details/ar_card_list.csv

```diff
@@ -0,0 +1,20 @@
+ar_ca0100#0,"""Beginner les『ARKS Road』"""
```

Comment:
https://github.com/Arks-Layer/PSO2ENPatchCSV/wiki/How-to-contribute#miscellanous-information

replace `『`and `』` with `'"`